migrations/20200203182043_fix_view_users_with_organisations.js

```diff
@@ -0,0 +1,13 @@
+exports.up = async function(knex) {
+    await knex.raw('DROP VIEW users_with_organisations')
+    await knex.raw(`
+        CREATE VIEW users_with_organisations  
+        AS SELECT u.id AS user_id, u.name as user_name, u.email, o.name AS organisation_name, o.id AS organisation_id FROM users AS u 
+            INNER JOIN user_organisation AS dorg 
+                ON dorg.user_id = u.id
+            INNER JOIN organisation AS o 
+                ON o.id = dorg.organisation_id
+        `)
+}
+
+exports.down = async function(knex) {}
```

Comment:
need to implement the down step as well

Reply:
Wouldn't this break stuff? It's already dropped in the rollback of the first migration that created the view. So while doing a rollback it would try to drop the table multiple times.

Reply:
This is a view and not a table, and it will break not having a rollback since then the database will contain a view that the code might not support

Reply:
The rollback should drop the view, and recreate the view as it was before being dropped in the up-state, e.g. just copy paste the previous closest migration creating the view, and then drop + paste that in the down step and you'll be fine